cosmo_tester/test_suites/image_based_tests/hello_world_test.py

```diff
@@ -13,32 +13,20 @@
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
 
-import pytest
-
-from cosmo_tester.framework.examples.hello_world import HelloWorldExample
-from cosmo_tester.framework.fixtures import image_based_manager
+from cosmo_tester.framework.fixtures import image_based_manager, hello_world  # noqa
```

Comment:
It shouldn't be necessary to explicitly import fixtures. If you don't then you shouldn't need the other  `# noqa`s either